src/xcode/ENA/ENA/Source/Models/Metadata/Test Result Metadata/__tests__/TestResultMetadataTests.swift

```diff
@@ -166,13 +172,13 @@ class TestResultMetadataTests: XCTestCase {
 		XCTAssertEqual(secureStore.testResultMetadata?.testResult, .positive, "testResult should be updated")
 	}
 
-	private func mockRiskCalculationResult(risk: RiskLevel = .high) -> RiskCalculationResult {
+	private func mockRiskCalculationResult(risk: RiskLevel = .high, mostRecentDateHighRisk: Date = Date(), mostRecentDateLowRisk: Date = Date()) -> RiskCalculationResult {
 		RiskCalculationResult(
 			riskLevel: risk,
 			minimumDistinctEncountersWithLowRisk: 0,
 			minimumDistinctEncountersWithHighRisk: 0,
-			mostRecentDateWithLowRisk: Date(),
-			mostRecentDateWithHighRisk: Date(),
+			mostRecentDateWithLowRisk: mostRecentDateLowRisk,
+			mostRecentDateWithHighRisk: mostRecentDateHighRisk,
```

Comment:
What would happen if `mostRecentDateLowRisk` < `mostRecentDateHighRisk`? Is there a test?